src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -56,6 +56,83 @@ We sometimes receive support requests on GitHub, ranging from questions about th
 
 There are some Issues which do not fit in the previous categories, such as [releases Issues](https://github.com/PrestaShop/PrestaShop/issues/20804). They serve a specific purpose.
 
+## Projects
+
+Available in the ‘[Projects](https://github.com/PrestaShop/PrestaShop/projects)’ tab of the PrestaShop’s GitHub repository, versions kanbans organize the Core team’s work. Both the product and the tech teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
+
+### Definition of states
+
+A ticket can either be an issue or a pull request. Most of the time, pull requests fix issues so it is most likely that you run into issues while browsing the kanban. A ticket should always have an owner, be it a PM or a developer.
```

Comment:
So both PR and issues are accepted in the kanban? I thought the purpose of this update was to avoid having PR in the kanban?

If exceptions are authorized we should define when a PR is acceptable, and maybe explain that an issue is always preferred when possible? and mandatory when it exists (only PR without issues are accepted?)

Reply:
You are right, I just edited this part here: https://github.com/PrestaShop/docs/pull/807/commits/03e8c670ae1de578f0c042976db69f746a31d9d4